<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_rpt_report_frame_dishsalerank</class>
 <widget class="QWidget" name="w_rpt_report_frame_dishsalerank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>65</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_group">
     <property name="text">
      <string>报表分组类型</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_group"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
